server/lib/util.js

```diff
@@ -235,6 +236,13 @@ function formatCsvToJson(filePath) {
   return defer.promise;
 }
 
+// calcualte age function
+function calcualteAge(dateString) {
+  const birthday = +new Date(dateString);
```

Comment:
Let's use `moment` for this.
```js
return moment().diff(dob, 'years');  
```

We already do this on the client, so it will make the presentation consistent between the two.

Reply:
okays, it more simple